src/test/java/uk/gov/mca/beacons/api/controllers/AccountHolderControllerUnitTest.java

```diff
@@ -35,6 +37,7 @@
 
 @WebMvcTest(controllers = AccountHolderController.class)
 @AutoConfigureMockMvc
+@Import(WebMvcTestConfiguration.class)
```

Comment:
Required due to `@WebMvcTest` not importing any `@Service` annotations which is required by the `RestResponseEntityExceptionHandler` as it injects the `GetValidationErrorResponseService`.

It does require us to add this import on any `WebMvcTest` as Spring will complain that the dependency cannot be found.

Another option I could think of was to:

- Create our own annotation that wraps `@Import(WebMvcTestConfiguration.class)` such as `@BeaconsMvcTestConfiguration`

I didn't see a huge benefit of doing this vs adding the annotated import on each `WebMvcTest` we create